### PR TITLE
[zh-CN] initial Notification.lang with en-US

### PR DESCRIPTION
**Pull request by skyclouds2001**
### Description

initial Notification.lang with en-US

### Motivation

initial Notification.lang with en-US

### Additional details

https://github.com/mdn/content/blob/main/files/en-us/web/api/notification/lang/index.md?plain=1

### CODE REVIEW

File: files/zh-cn/web/api/notification/lang/index.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+{{APIRef("Web Notifications")}}{{AvailableInWorkers}}{{securecontext_header}}
+
+{{domxref("Notification")}} 接口的 **`lang`** 只读属性指示通知中使用的语言，在 {domxref("Notification.Notification","Notification()")}} 构造函数的 `lang` 参数中指定。
+
+语言本身是根据 {{RFC(5646, "Tags for Identifying Languages (also known as BCP 47)")}} 使用表示语言标签的字符串指定的。请参阅 Sitepoint [ISO 2 字母语言代码](https://www.sitepoint.com/iso-2-letter-language-codes/) 页面以获取简单参考。
+
+## 值
+
+一个指定语言标记的字符串。
```
Comment (skyclouds2001) [inline files/zh-cn/web/api/notification/lang/index.md:14]:
```suggestion
一个指定语言标签的字符串。
```

File: files/zh-cn/web/api/notification/lang/index.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+## 示例
+
+以下代码段会触发一条通知；创建一个简单的 `options` 对象，然后使用 `Notification()` 构造函数触发通知。
```
Comment (skyclouds2001) [inline files/zh-cn/web/api/notification/lang/index.md:18]:
```suggestion
以下代码段会触发一条通知；首先创建一个简单的 `options` 对象，然后使用 `Notification()` 构造函数触发通知。
```

File: files/zh-cn/web/api/notification/lang/index.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+```js
+const options = {
+  body: "Your code submission has received 3 new review comments.",
```
Comment (skyclouds2001) [inline files/zh-cn/web/api/notification/lang/index.md:22]:
```suggestion
  body: "您提交的代码收到了 3 条新的审阅意见。",
```

File: files/zh-cn/web/api/notification/lang/index.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+  lang: "en-US",
+};
+
+const n = new Notification("New review activity", options);
```
Comment (skyclouds2001) [inline files/zh-cn/web/api/notification/lang/index.md:26]:
```suggestion
const n = new Notification("新的评论活动", options);
```

File: files/zh-cn/web/api/notification/lang/index.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+
+```js
+const options = {
+  body: "您提交的代码收到了 3 条新的审阅意见。",
+  lang: "en-US",
+};
+
+const n = new Notification("新的评论活动", options);
+
+console.log(n.lang); // "en-US"
```
Comment (skyclouds2001) [inline files/zh-cn/web/api/notification/lang/index.md:28]:
```suggestion
console.log(n.lang); // "zh-Hans-CN"
```

File: files/zh-cn/web/api/notification/lang/index.md [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+一个指定语言标签的字符串。
+
+## 示例
+
+以下代码段会触发一条通知；首先创建一个简单的 `options` 对象，然后使用 `Notification()` 构造函数触发通知。
+
+```js
+const options = {
+  body: "您提交的代码收到了 3 条新的审阅意见。",
+  lang: "en-US",
```
Comment (skyclouds2001) [inline files/zh-cn/web/api/notification/lang/index.md:23]:
```suggestion
  lang: "zh-Hans-CN",
```

**Review by skyclouds2001 — COMMENTED** [review]
see [rfc5646](https://datatracker.ietf.org/doc/html/rfc5646#appendix-A)

File: files/zh-cn/web/api/notification/lang/index.md [hunk 7]
```diff
@@ -0,0 +1,41 @@
+---
+title: Notification：lang 属性
+slug: Web/API/Notification/lang
+---
+
+{{APIRef("Web Notifications")}}{{AvailableInWorkers}}{{securecontext_header}}
+
+{{domxref("Notification")}} 接口的 **`lang`** 只读属性指示通知中使用的语言，在 {domxref("Notification.Notification","Notification()")}} 构造函数的 `lang` 参数中指定。
```
Comment (yin1999) [inline files/zh-cn/web/api/notification/lang/index.md:8]:
```suggestion
{{domxref("Notification")}} 接口的 **`lang`** 只读属性指示在 {domxref("Notification.Notification","Notification()")}} 构造函数的 `lang` 参数中指定的通知中使用的语言。
```

File: files/zh-cn/web/api/notification/lang/index.md [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+
+语言本身是根据 {{RFC(5646, "Tags for Identifying Languages (also known as BCP 47)")}} 使用表示语言标签的字符串指定的。请参阅 Sitepoint [ISO 2 字母语言代码](https://www.sitepoint.com/iso-2-letter-language-codes/) 页面以获取简单参考。
```
Comment (yin1999) [inline files/zh-cn/web/api/notification/lang/index.md:10]:
```suggestion
语言本身是根据 {{RFC(5646, "Tags for Identifying Languages（即 BCP 47）")}}使用表示语言标签的字符串指定的。请参阅 Sitepoint [ISO 2 字母语言代码](https://www.sitepoint.com/iso-2-letter-language-codes/) 页面以获取简单参考。
```

**Review by yin1999 — APPROVED** [review]
Thanks

**Pull request merged**